test/contrib/test_pyopenssl.py

```diff
@@ -55,3 +57,13 @@ def test_dnsname_to_stdlib_leading_splat(self):
         expected_result = '*.xn--p1b6ci4b4b3a.xn--11b5bs8d'
 
         self.assertEqual(_dnsname_to_stdlib(name), expected_result)
+
+    def test_inject_validate_fail(self):
+        """
+        Injection should not be supported if we are missing required dependencies
+        """
+        with patch("cryptography.x509.extensions.Extensions") as mock:
+            del mock.get_extension_for_class
+            with self.assertRaises(ImportError):
+                inject_into_urllib3()
```

Comment:
This test passes for me locally. At least, I'm pretty sure it does. I actually get a lot of failures when I try to run this suite locally, though I don't think this test is among them. How do y'all run the test suite locally? Or do you just always use Travis-CI?